lib/common/constants.js

```diff
@@ -26,7 +26,7 @@ function constantsFactory (path) {
               error: 'red',
               warning: 'yellow',
               info: 'green',
-              debug: 'blue',
+              debug: 'cyan',
```

Comment:
Heh, so, I can tell the difference between cyan and the other colors, however I **did** notice that I actually can't tell the difference between info and warning messages (green and yellow)!